lib/lookup.json

```
@@ -24,6 +24,13 @@
     "prefix": "v",
     "flaky": true
   },
+  "@yarnpkg/cli": {
+    "maintainers": "arcanis",
+    "yarn": true,
+    "prefix": "@yarnpkg/cli@",
+    "scripts": ["typecheck:all", "test:lint", "test:unit", "build:cli"],
```

```suggestion
    "scripts": ["build:cli", "typecheck:all", "test:lint", "test:unit", "test:integration"],
```
Most of our tests are integration tests and are run by `test:integration` which requires `build:cli` to run first.

---

@merceyz it will take much time to run "test:integration" and probably timeout

---

We mostly have integration tests, so running just the unit tests wont catch much.
On a Ubuntu 20.04 GitHub Actions runner it takes [about 22 minutes](https://github.com/yarnpkg/berry/runs/6352041638) to run. I see [Jest has a timeout of 30 minutes](https://github.com/nodejs/citgm/blob/908ebe2beac091131a8dbb73af11b55cf424add1/lib/lookup.json#L269) so assuming we're granted the same timeout that should be fine.